sections/1-introduction.tex

```diff
@@ -3,28 +3,33 @@ \section{Introduction}
-As such, new members of the solar physics community are less likely to need to learn a new programming language to become productive researchers and can benefit from the wealth of \python documentation and tutorials available online.
-
-The primary motivating factor for choosing \python is the rich and mature ecosystem of packages for performing scientific analysis and computation. 
+The choice of \python to develop analysis tools for the community was motivated by several different factors.
+The primary motivating factor is the rich and mature ecosystem of packages for performing scientific analysis and computation. 
 The scientific \python ecosystem is supported by foundational packages for manipulating tabular \citep[\pandas,][]{pandas} and multi-dimensional array\citep[\numpy,][]{numpy} data, general purpose scientific computing \citep[\scipy,][]{scipy}, and publication-quality 2D plotting \citep[\matplotlib,][]{matplotlib}. 
 These core packages form the backbone of hundreds of additional scientific \python packages, such as \astropypkg for functionality and tools specific to astronomy, scikit-learn for machine learning and data mining \citep{pedregosa11}, and Dask for parallel and distributed computing \citep{rocklin15}. Interoperability between all these packages enables interdisciplinary analysis across solar physics, space physics, and astrophysics as well as the greater scientific community.
-Notably, packages from the scientific Python ecosystem played a key role in the first detection of a gravitational wave \citep{ligo_scientific_collaboration_and_virgo_collaboration_observation_2016} as well as the recent imaging of a supermassive black hole \citep{collaboration_first_2019}.
+
+Several other cultural factors motivated the adoption of \python.
+The Python programming language is freely-available and open-source, meaning users are not bound by restrictive and costly proprietary licenses.
+Python is one of the most widely-used programming languages by professional software developers\footnote{According to the 2019 Stack Overflow developer survey(\url{https://insights.stackoverflow.com/survey/2019}), Python is the fourth most popular language among professional developers.} and is also now used by most universities to teach computer science \citep{guo2014}.
+Therefore, early career members of the solar physics community will likely already know Python.
+The scientific \python ecosystem also adopted a culture of community development, which means that the entire community is welcome to openly develop functionality to either enhance existing packages like SunPy or create new ones (see Section \ref{sec:development}).
```

Comment:
I'd suggest removing  "scientific." While true, I think this system of open development/community engagement is really due to the culture of the Python community as a *whole*, not just scientific Python.